cross/libev/Makefile

```diff
@@ -2,7 +2,7 @@ PKG_NAME = libev
 PKG_VERS = 4.22
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://dist.schmorp.de/$(PKG_NAME)
+PKG_DIST_SITE = http://dist.schmorp.de/$(PKG_NAME)/Attic
```

Comment:
Is is ok to use the Attic sub folder, since it holds all public releases. As far as the version and the digests file does not change it does not have any side effects.

Reply:
Fair enough, just wanted to isolate this from the contents of the PR itself.
I removed it anyway.
(Will create another PR right away then ;) )

Reply:
Moved it there: https://github.com/SynoCommunity/spksrc/pull/3785